src/components/shared/Entity/Create/index.tsx

```diff
@@ -186,7 +186,7 @@ function EntityCreate({
             </Collapse>
 
             {!isValidating && connectorTags.length === 0 ? (
-                <AlertBox severity="warning">
+                <AlertBox short={false} severity="warning">
```

Comment:
I have marked `short' was required. That way we can change the default to true soon and then mark it as optional again.